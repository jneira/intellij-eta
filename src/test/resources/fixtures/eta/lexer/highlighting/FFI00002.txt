ITinline_prag ('{-#')
ITline_prag (' LANGUAGE ForeignFunctionInterface, InterruptibleFFI, CApiFFI ')
ITline_prag ('#-}')
WHITE_SPACE ('\n')
ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('FFI00002')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITforeign ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('interruptible')
WHITE_SPACE ('\n   ')
ITstring ('"')
ITstring ('sleep')
ITstring ('"')
WHITE_SPACE (' ')
ITvarid ('sleep')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CUint')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CUint')
WHITE_SPACE ('\n\n')
ITforeign ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('safe')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('wool-common.h wool_init')
ITstring ('"')
WHITE_SPACE (' ')
ITvarid ('initWool')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE ('\n')
ITforeign ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('jvm')
WHITE_SPACE (' ')
ITvarid ('interruptible')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('wool-common.h wool_init2')
ITstring ('"')
WHITE_SPACE (' ')
ITvarid ('initWool2')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')
