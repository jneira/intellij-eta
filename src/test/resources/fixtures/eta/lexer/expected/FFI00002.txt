ITblockComment ('{-# LANGUAGE ForeignFunctionInterface, InterruptibleFFI, CApiFFI #-}')
WHITE_SPACE ('\n')
ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('FFI00002')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('interruptible')
WHITE_SPACE ('\n   ')
ITstring ('"sleep"')
WHITE_SPACE (' ')
ITvarid ('sleep')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CUint')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CUint')
WHITE_SPACE ('\n\n')
ITsemi ('')
WHITE_SPACE ('')
ITvarid ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('safe')
WHITE_SPACE (' ')
ITstring ('"wool-common.h wool_init"')
WHITE_SPACE (' ')
ITvarid ('initWool')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE ('\n')
ITsemi ('')
WHITE_SPACE ('')
ITvarid ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('jvm')
WHITE_SPACE (' ')
ITvarid ('interruptible')
WHITE_SPACE (' ')
ITstring ('"wool-common.h wool_init2"')
WHITE_SPACE (' ')
ITvarid ('initWool2')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')